addon/ng2/commands/github-pages-deploy.ts

```diff
@@ -26,11 +27,16 @@ module.exports = Command.extend({
       type: String,
       default: 'new gh-pages version',
       description: 'The commit message to include with the build, must be wrapped in quotes.'
-    }, {
+    }, { 
+      name: 'target',
+      type: String,
+      default: 'development', 
```

Comment:
This differs to the old behaviour, where the defaut was `production`! Old setting makes much more sence, since gh-pages is a production environment.